caab-service/src/main/java/uk/gov/laa/ccms/caab/api/entity/CaseOutcome.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.laa.ccms.caab.api.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+/**
+ * Represents a case outcome entity associated with the "XXCCMS_CASE_OUTCOME" table.
+ *
+ * <p>This entity is utilized to manage and persist case outcome data
+ * within the CCMS system. It makes use of the "XXCCMS_GENERATED_ID_S"
+ * sequence for generating unique identifiers.</p>
+ */
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+@Table(name = "XXCCMS_CASE_OUTCOME")
+@SequenceGenerator(
+    allocationSize = 1,
+    sequenceName = "XXCCMS_GENERATED_ID_S",
+    name = "XXCCMS_CASE_OUTCOME_S")
+@Getter
+@Setter
+@RequiredArgsConstructor
+public class CaseOutcome {
+
+  @Id
+  @GeneratedValue(generator = "XXCCMS_CASE_OUTCOME_S")
+  private Long id;
+  /**
+   * audit trail info.
+   */
+  @Embedded
+  private AuditTrail auditTrail = new AuditTrail();
+
+  @Column(name = "LSC_CASE_REFERENCE", length = 50, nullable = false)
```

Comment:
Could this be changed to a foreign key/back ref to an Application?

Reply:
Case outcome will be joined to multiple entities, such as cost awards